.github/workflows/release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       image: docker.mirror.hashicorp.services/hashicorp/jsii-terraform
     env:
       CHECKPOINT_DISABLE: "1"
-
+      DISABLE_VERSION_CHECK: "1"
```

Comment:
While this unblocks us in the short term, I'd rather not do this and try to figure out and fix the underlying issue.